Ni236630/Glassdale
# Nd officer select

## Changes
1) Created custom event for officer select
2) Added event hub in criminalList

## Test
1) serve up on your favorite port 
2) check both criminalList.js and officerSelect.js that both have events listeners
3) check browser for filtering
